README.md

```diff
@@ -269,18 +269,34 @@ URLs which host secrets must be available every time a request is executed by DO
+
+3. Encode the request parameters into CBOR and store it on chain with `npx hardhat functions-set-auto-request --network network_name_here  --subid subscription_id_number_here --interval time_between_requests_here --contract 0x_contract_address`.  You can now manually check that your on-chain requests work if you try the manual debugging step #4 or you can go ahead and run Chainlink Automations on your contract by following the next step.  
+
+> ⚠️⚠️⚠️ Keep in mind that this task sets DON-Hosted secrets and expires those secrets after 10 minutes.  If you see error bytes returned to your client which decode to secrets not being found, you should run this functions-set-auto-request command or the functions-upload-secrets-don to refresh the DON-hosted secrets for that slotId.
+
+4. Register the contract for upkeep via the Chainlink Automation web app here: [https://automation.chain.link/](https://automation.chain.link/)
    - Be sure to set the `Gas limit` for the _performUpkeep_ function to a high enough value. The recommended value is 1,000,000.
    - Find further documentation for working with Chainlink Automation here: [https://docs.chain.link/chainlink-automation/introduction](https://docs.chain.link/chainlink-automation/introduction)
 
-Once the contract is registered for upkeep, check the latest response or error with the commands `npx hardhat functions-read --network network_name_here --contract contract_address_here`.
+Once the contract is registered for upkeep, check the latest response or error with the commands `npx hardhat functions-read --network network_name_here --contract contract_address_here`.  Note that you can also pause and unpause your Automation Upkeep using the web app's UI.
+
+4. For debugging, use the command `npx hardhat functions-check-upkeep --network network_name_here --contract contract_address_here` to see if Automation needs to call _performUpkeep_. If this call returns `false` then the upkeek interval has not yet passed and `performUpkeep` will not execute.
+
+You can also attach a listener to a Subscription ID by updating the `network` and `subId` variables in `listen.js` and then running `node listen.js` from the repo root. 
+
+Once the listener is running (and after each update interval), you can also manually trigger a request by using the command `npx hardhat functions-perform-upkeep --network network_name_here --contract contract_address_here`. If your DON hosted secrets have not been expired and the JS source does not throw errors during execution, you will see the result of the computation printed to your console.
```

Comment:
This line is just a bit confusing as the listener does not need to be running in order to perform a manual upkeep.
TBH, I think we could just remove this line altogether as functions-perform-upkeep should only ever need to be used if Automation is broken...
In fact, functions-perform-upkeep likely won't even work because the Automation nodes will likely perform the upkeep before a user even has time to run the CLI command.